bin/bin.js

```diff
@@ -355,6 +357,10 @@ const main = async () => {
 
     nextItem();
   }
+  const dlEpisodeText = dlCounter === 1 ? "episode" : "episodes";
```

Comment:
Two quick things:

- I think we can get away without adding a log message because there is a lot of logging for each episode.
- I would like to reserve exit code `1` for traditional errors. Could we switch it to `2`?

Something like below would handle both items:

```js
if (episodesDownloadedCounter === 0) {
  process.exit(2);
}
```

Reply:
Sure, I agreed it's better to have another exit code than 1. So resolved by e0fd67d